pkg/constants/constants.go

```diff
@@ -227,8 +226,6 @@ var ResourcesToBlock = map[string]bool{
 	"networkinterfaces.netoperator.vmware.com":                true,
 	"networks.netoperator.vmware.com":                         true,
 	"nsxerrors.nsx.vmware.com":                                true,
-	"nsxlbmonitors.vmware.com":                                true,
-	"nsxloadbalancermonitors.vmware.com":                      true,
```

Comment:
Same here.  Add the following and explain that we need these in restore skip list, but don't add to backup skip list here.

//	"nsxlbmonitors.vmware.com":                                true,
//	"nsxloadbalancermonitors.vmware.com":                      true,

Reply:
Done